src/transports/dash/manifest_parser.ts

```diff
@@ -153,7 +153,7 @@ export default function generateManifestParser(
           return Promise.reject(cancelSignal.cancellationError);
         }
         const warnings: IPlayerError[] = [];
```

Comment:
From what I understand, `warnings` isn't used anywhere anymore? We could remove it from the `transports` API